app/models/like.rb

```diff
@@ -1,4 +1,15 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :increment_post_likes_counter
+  after_destroy :decrement_post_likes_counter
+
+  def increment_post_likes_counter
+    post.increment!(:likes_counter)
+  end
+
+  def decrement_post_likes_counter
+    post.decrement!(:likes_counter)
+  end
```

Comment:
[OPTIONAL] Great work for using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html). There's a better way with lesser line of code.
- You can use `after_save` to call the `#update_like_counter` method every time a new like has been saved to database, then make the `update_like_counter` method private.
-  It should look something similar to this. :point_down: 

```
 # app/models/like.rb
  after_save :update_like_counter

  private

  def update_like_counter
    post.increment!(:likes_counter)
  end
```